src/Zadatak2/Rectangle.java

```diff
@@ -0,0 +1,46 @@
+package Zadatak2;
+
+public class Rectangle {
+    //attributes
+    private double side_a;
+    private double side_b;
+
+    //constructor
+    public Rectangle(final double side_a, final double side_b){
+        this.side_a = side_a;
+        this.side_b = side_b;
+    }
+
+    //getters
+    public double getSide_a(){
+        return this.side_a;
+    }
+    public double getSide_b() {
+        return side_b;
+    }
+
+    //setters
+    public void setSide_a(final double side_a){
+        this.side_a = side_a;
+    }
+    public void setSide_b(final double side_b){
+        this.side_b = side_b;
+    }
+
+    //methods
+    public double getCircumference(){
+        return (2*side_a) + (2*side_b);
+    }
+    public double getArea(){
+        return side_a*side_b;
+    }
+    public boolean isSquare(){
+        if (side_b == side_a){
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
```

Comment:
`side_b == side_a` evaluates to `boolean`, thus this could have been simplified as:
`return side_b == side_a;`

Reply:
I get it but not fully. If we return `side_a == side_b` how do we know it is true? Unless true is a default?

Reply:
the equation `side_a == side_b` is a `boolean`. It will have the value `true` if both sides are equal, try it in code.